### PR TITLE
Update and move release process

### PR DESCRIPTION
**Pull request by seanbudd**
### Summary of the issue:
NV Access is intending to move most of the wiki into the repository, so changes are tracked better and easier to propose.
The release process wiki page is a prime candidate for migration.
The release process wiki is out of date, it also contains some information that is better suited to or duplicated in other documents.

### Description of user facing changes
[The release process wiki page](https://github.com/nvaccess/nvda/wiki/ReleaseProcess) is migrated to the repository.

The process is updated to include more information on the timing of the stages.
Information for testers, and developers are moved to the appropriate contributing docs.

- [x] On merge: update wiki page, index and sidebar to forward to project docs

### CODE REVIEW

File: projectDocs/dev/contributing.md [hunk 1]
```diff
@@ -45,6 +46,9 @@ If you are new to the project, or looking for some way to help take a look at:
 	If these fail, please review them.
 	Sometimes system tests fail unexpectedly.
 	If you believe the failure is unrelated, feel free to ignore it unless it is raised by a reviewer.
+	- All pull requests submitted must have their "Allow edits from maintainers" checkbox ticked.
+	This is the GitHub default for new pull requests.
+	- A lead developer may specifically requested the pull request be made against `beta` or `rc` in the case of addressing bugs introduced in the current release cycle.
```
Comment (michaelDCurran) [inline projectDocs/dev/contributing.md:51]:
```suggestion
	- A lead developer may specifically request the pull request be made against `beta` or `rc` in the case of addressing bugs introduced in the current release cycle.
```

File: projectDocs/community/releaseProcess.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+ 
+## Release Workflow
+This is the general release workflow. Information for specific community groups is provided in later sections. The production of a release consists of the following:
+1. [Alpha Phase](#alpha-phase) (~7 weeks)
+   - Development is done in parallel to the release process for the prior version.
+   - e.g. NVDA 2020.2 is in alpha while NVDA 2020.1 is going through the Release Phase
+1. [Release Phase](#release-phase) (~7 weeks)
+   - [Beta1](#beta-builds) 1 week after the most recent final release
+   - A new beta is released weekly as required
+   - Once a week passes with a stable beta, the 2 week [translation freeze begins](#translatable-string-freeze)
```
Comment (michaelDCurran) [inline projectDocs/community/releaseProcess.md:16]:
```suggestion
   - Once a beta has been stable for one week (no issues reported), the 2 week [translation freeze begins](#translatable-string-freeze)
```

File: projectDocs/community/releaseProcess.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+   - Once the translation freeze ends, [a release candidate (RC)](#release-candidate) is created
+   - A new RC is released weekly as required
+   - Once a week passes with a stable RC, the final release is created
```
Comment (michaelDCurran) [inline projectDocs/community/releaseProcess.md:19]:
```suggestion
   - when an RC has been stable for one week (no issues reported), the final release is created
```

File: projectDocs/community/releaseProcess.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+#### Beta builds
+* A commit without any known serious issues, will be selected from the 'master' branch and merged into 'beta', this draws the line for features included in the release.
+* Documentation changes will be reviewed. A release summary will be added to the change log for the beta.
+* A tagged 'beta release' will be created for wider testing. 
+* New pull requests may be now considered for squash merging straight to beta.
+  - If addressing regression introduced in this release.
+  - If addressing a bug in a "must have" feature for this release.
+  - If addressing a critical Operating System change out of our control.
+* As appropriate new tagged beta releases will be published.
```
Comment (michaelDCurran) [inline projectDocs/community/releaseProcess.md:48]:
```suggestion
* As appropriate new tagged beta releases will be published once a week.
```

**Review by michaelDCurran — CHANGES_REQUESTED** [review]

**Review by michaelDCurran — APPROVED** [review]

**Review by Qchristensen — APPROVED** [review]
Reads well, good work.

**Pull request merged**